recipe/meta.yaml

```diff
@@ -53,14 +53,15 @@ requirements:
     - rich >=10.13.0
     - pyparsing >=2.4.7
     - typing_extensions >=3.7.4
-    - scmrepo ==0.1.4
-    - dvc-render ==0.0.15
+    - scmrepo ==0.1.5
+    - dvc-render ==0.0.17
     - dvclive >=1.2.2
-    - dvc-data ==0.28.4
-    - dvc-http ==2.27.2
-    - dvc-task ==0.1.8
+    - dvc-data ==0.28.5
+    - dvc-http
+    - dvc-task ==0.1.9
     - hydra-core >=1.1.0
     - iterative-telemetry ==0.0.6
+    - dvc-studio-client >=0.1.1
```

Comment:
The [`dvc-studio-client`](https://github.com/conda-forge/dvc-studio-client-feedstock/tree/6ac1571f8d3a725916390699eda133f2f100c738) conda-forge package has incorrect version information, resulting in this error:

```
dvc 0.0.0 has requirement dvc-studio-client>=0.1.1, but you have dvc-studio-client 0.0.0.
```

I'll submit a patch to https://github.com/conda-forge/dvc-studio-client-feedstock to fix this. It requires some modification to the `recipe/meta.yaml` file because `dvc-studio-feedstock` uses setuptools_scm.